src/Momento.Sdk/Config/Retry/FixedTimeoutRetryStrategy.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using Grpc.Core;
+using Microsoft.Extensions.Logging;
+using Momento.Sdk.Internal.Retry;
+
+namespace Momento.Sdk.Config.Retry;
+
+/// <summary>
+/// Retry failed requests up until the client's timeout has been reached.
+/// Retries are attempted at fixed intervals (retryDelayIntervalMillis) +/- jitter.
+/// Retry attempts time out after responseDataReceivedTimeoutMillis to be able to 
+/// allow multiple retries before the client's timeout is reached.
+/// </summary>
+public class FixedTimeoutRetryStrategy : IRetryStrategy
+{
+    private ILoggerFactory _loggerFactory;
+    private ILogger _logger;
+    private readonly IRetryEligibilityStrategy _eligibilityStrategy;
+    private readonly TimeSpan _retryDelayIntervalMillis;
+    private readonly TimeSpan _responseDataReceivedTimeoutMillis;
+
+    /// <summary>
+    /// Constructor for the FixedTimeoutRetryStrategy.
+    /// </summary>
+    /// <param name="loggerFactory"></param>
+    /// <param name="eligibilityStrategy"></param>
+    /// <param name="retryDelayIntervalMillis">Amount of time between retry attempts.</param>
+    /// <param name="responseDataReceivedTimeoutMillis">How long to wait for a retry attempt to succeed or timeout.</param>
+    public FixedTimeoutRetryStrategy(ILoggerFactory loggerFactory, IRetryEligibilityStrategy? eligibilityStrategy = null, TimeSpan? retryDelayIntervalMillis = null, TimeSpan? responseDataReceivedTimeoutMillis = null)
```

Comment:
When using `TimeSpan` objects, you should remove the `Millis` suffix, since a `TimeSpan` allows for building using whatever time unit (or units) you want.